process/io/plot_proc.py

```diff
@@ -2196,6 +2199,54 @@ def plot_current_drive_info(axis, mfile_data, scan):
             data[0] = ((pinjie - pinjmwfix), "Primary auxiliary power", "MW")
             data.insert(2, (pinjie, "Total auxillary power", "MW"))
 
+    if lhcd:
+        data = [
+            (pinjie, "Steady state auxiliary power", "MW"),
+            ("pheat", "Power for heating only", "MW"),
+            ("bootipf", "Bootstrap fraction", ""),
+            ("faccd", "Auxiliary fraction", ""),
+            ("facoh", "Inductive fraction", ""),
+            ("powerht", "Plasma heating used for H factor", "MW"),
```

Comment:
in the NBI output `gamnb` is printed. We should include the normalised current drive efficiency in the other outputs. Can we add a line for `gamcd` here? Also add this in the ERCH and EBW cases.

Reply:
I added this in and also for ECRH and EBW